apps/next/components/user/login-form.tsx

```diff
@@ -33,7 +34,31 @@ import { Switch } from "@/components/ui/switch"
 
 import { signInWithEmailPassword, signInWithOtp } from "@/modules/user/auth"
 
-export const LoginForm: React.FC = () => {
+const getErrorProps = (
```

Comment:
Could you please use the function keyword for functions.

Reply:
Good to know. Didn't notice at first. Could be added to contribution guide.